modules/ROOT/pages/lb-cert-validation.adoc

```diff
@@ -81,7 +80,7 @@ AES128-SHA
 DES-CBC3-SHA
 ----
 
-ClourHub's dedicated load balancer supports TLSv1.1 and TLSv1.2. Additionally, you can configure TLS v1.0. However, TLS v1.0 is no longer accepted by PCi compliance due to its significant vulnerabilities.
+CloudHub's dedicated load balancer supports TLSv1.1 and TLSv1.2. You can configure TLS v1.0., but TLS v1.0 is no longer accepted by PCI compliance due to its significant vulnerabilities.
```

Comment:
Space or no space after TLS and before the version number? Both are used here.

I see THREE versions online: no space, space, and no "v" but a space (TLS 1.0).

I need to confer with team and create a style guideline about this picky little thing....